packages/wrangler/src/index.tsx

```diff
@@ -229,6 +229,10 @@ export async function main(argv: string[]): Promise<void> {
               {
                 name: "worker",
                 version: "0.0.0",
+                scripts: {
+                  start: "wrangler dev src/index.ts",
```

Comment:
This will be wrong if typescript isn't chosen.

Reply:
I was thinking about that as well. My thought is to have this done conditionally based on that user choice.